### ISSUE TITLE
Pkg.jl will corrupt itself on Windows

### ISSUE THREAD

**Issue by Plecra** [fonsp/Pluto.jl#671]
As I see it, https://github.com/JuliaLang/Pkg.jl/issues/1667 is a high priority for Pluto. If we want to create an accessible programming environment for beginners, it's pretty important that Pluto doesn't lock up without any explanation. And we definitely don't want the explanation to be "you must reinstall the whole environment every time you want a new package".

**Comment by fonsp** [comment]
Thanks! What should we do about it? The Pkg developers will definitely solve this problem, but it might take a while.

**Comment by Plecra** [comment]
Documentation explaining that users may need to add an AV exception would be a short term fix.
Beyond that, I think having an issue open about it will help guide Pluto users to the discussion and hopefully help identify the exact issue.

**Comment by fonsp** [comment]
I agree. Can you describe when the corruption happens normally? Is it in cells like:
```julia
begin
	import Pkg
	Pkg.activate(mktempdir())
	Pkg.add(["Plots", "DataFrames"])
	using Plots
	using DataFrames
end
```
or does it happen when you install packages using the `pkg> ` REPL command line, before opening Pluto?

**Comment by Plecra** [comment]
It's been happening cells like

```jl
using Plots
```

Before adding the exception to defender, I found that this resulted in 

- The notebook seemingly get stuck on loading the cell
- The log just hanging at some point while installing the package (I left this for ~15m, much longer than the installation usually took)
- When cancelled, the registry was corrupted and needed to be cleared

**Comment by fonsp** [comment]
Are you sure? `using Plots` does not do any `Pkg` operations

**Comment by Plecra** [comment]
It will automatically install `Plots` if the package is missing

**Comment by fonsp** [comment]
That's not true

**Comment by Plecra** [comment]
Whoops >.< I assume this was happening in the sample notebooks which use the `Pkg` API then

**Comment by fonsp** [comment]
Generally solved because of #844 and https://julialang.org/blog/2021/11/julia-1.7-highlights/#improved_performance_for_handling_registries_on_windows_and_distributed_file_systems
